packages/components/src/Table/_docs/Table.stickersheet.stories.tsx

```diff
@@ -0,0 +1,701 @@
+import React from "react"
```

Comment:
It's not technically using the sticker sheet since we'd be fighting against the styles, but this is still used to show the many states so I figured I'd keep with the parlance